docs/book/installation/sylius_plus_installation.rst

```diff
@@ -345,7 +347,14 @@ your application's ``.env`` file:
     yarn build
     bin/console assets:install --ansi
 
-**11.** For more details check the installation guides for all plugins installed as dependencies with Sylius Plus.
+**11.** Clear cache for proper display of all translations:
```

Comment:
I would call it ` Rebuild cache` as just after clear we are also warming it up 😄